cartography/intel/okta/oktaintel.py

```diff
@@ -0,0 +1,1193 @@
+# Okta intel module
+import json
+import logging
+import os
+
+from okta import AppInstanceClient
+from okta import FactorsClient
+from okta import UserGroupsClient
+from okta import UsersClient
+from okta.framework.ApiClient import ApiClient
+from okta.framework.OktaError import OktaError
+from okta.framework.PagedResults import PagedResults
+from okta.models.usergroup import UserGroup
+
+from cartography.config import Config
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+OKTA_API_KEY = os.environ.get('CREDENTIALS_OKTA_API_KEY')
+
+OKTA_ORG_ID = os.environ.get("OKTA_ORG_ID")
```

Comment:
These pieces of configuration should not be read directly from the environment like this. They should be added to the `Config` class, which is initialized by cartography and passed to all intel modules. There are currently no docs covering this -- we can sync on it in person.